### ISSUE TITLE
Not Registering Custom Tags

### ISSUE THREAD

**Issue by jaman1020** [livingsocial/swagger_yard#59]
It looks like `SwaggerYard.register_custom_yard_tags!` is never fired when initializing `SwaggerYard`. Before adding this line to my own config I got errors like `Unknown tag @resource in file` and the documentation is never generated. Is this an oversight or am I doing something wrong? I've followed the docs pretty much exactly

**Comment by nicksieger** [comment]
You're correct about this. Usually we've used this in conjunction with swagger_yard-rails which does call that initializer. If you think it would be better to call it automatically, would you mind submitting a PR for that?
